app/views/alchemy/admin/attachments/_tag_list.html.erb

```diff
@@ -7,7 +7,7 @@
 <% if p[:tagged_with].present? %>
   <%= link_to(
     render_icon('delete-small') + _t('Remove tag filter'),
-    url_for(p.delete_if { |k, v| k == "tagged_with" }.merge(action: 'index')),
+    resource_url_proxy.url_for(p.delete_if { |k, v| k == "tagged_with" }.merge(action: 'index')),
```

Comment:
This and the other `_tag_list.html.erb` change aren't required, but I added them anyway for consistency, so anyone who ends up making a new `_tag_list` for another admin section doesn't run into the same problem I did.